README.md

```diff
@@ -130,13 +130,33 @@ developers monitor both channels.
 
 ### News and Updates 📣
 
+- **Feb 2024**: Version 0.14 is released, which deprecates [jj checkout and merge](changelog.md#deprecations),
+  and `jj init --git`, which is now just called `jj git init`. 
+- **Feb 2024**: Chris Krycho published an article about Jujutsu called [jj init][krycho]
+  and Steve Klabnik followed up with the [Jujutsu Tutorial][klabnik]. 
+- **Jan 2024**: Jujutsu was featured in an LWN.net article, called 
```

Comment:
nit: drop the comma? I find it easier to read without it, and you don't have a comma before "called" on the line above

Reply:
done